shopfloor/actions/move_line_search.py

```diff
@@ -104,7 +104,9 @@ def _sort_key_move_lines(order):
 
     def counters_for_lines(self, lines):
         # Not using mapped/filtered to support simple lists and generators
-        priority_lines = [x for x in lines if x.picking_id.priority == "1"]
+        priority_lines = [
+            x for x in lines if ord((x.picking_id.priority or "0")[0]) > 48
+        ]
```

Comment:
```suggestion
        priority_lines = [
            x for x in lines if int(x.picking_id.priority or "0") > 0
        ]
```
isn't this more clear?

Reply:
It is, but less resilient if someone decides to create a priority with the first char not being a number.

Reply:
just compare the strings then ? They are ordered:
```python
>>> "1" > "0"
True
>>> "a" > "0"
True
```

Reply:
@TDu I'm not sure it's something that will happen, it is expected to have a number here, at least a negative one but `int` will still handle it.

Reply:
Updated the code with @simahawk solution.